src/util/yp_string.c

```diff
@@ -125,7 +125,8 @@ yp_string_mapped_init(yp_string_t *string, const char *filepath) {
     // the source to a constant empty string and return.
     if (file_size == 0) {
         CloseHandle(file);
-        yp_string_mapped_init_internal(string, "", 0);
```

Comment:
CRuby warns that `""` is a `const char[1]` not a `const char*`

Reply:
What if we make `yp_string_mapped_init_internal` take `const char*`? Currently it takes `char*`